app/controllers/sessions_controller.rb

```diff
@@ -1,21 +1,30 @@
 class SessionsController < ApplicationController
 
   def new
-    
+    @user = User.new
   end
 
   def create
-    user = User.find_by(email: params[:session][:email])
-    if user && user.authenticate(params[:session][:password])
-      if user.user?
-        redirect_to user_path(user) # flash[:notice] = "You are currently logged in as #{user.full_name}."
-      elsif user.admin?
-        redirect_to '/admin' # flash[:notice] "Logged in."
+    @user = User.find_by(email: params[:session][:email])
+    if @user && @user.authenticate(params[:session][:password])
+      if @user.user?
+        session[:user_id] = @user.id
+        # flash[:notice] = "You are currently logged in as #{user.full_name}."
+        redirect_to root_path
+      elsif @user.admin?
+        flash[:notice] = "Admin logged in."
+        redirect_to '/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.